scm/etc/scripts/extract_FV3GFS_column_ic.py

```diff
@@ -0,0 +1,413 @@
+#!/usr/bin/env python
```

Comment:
This script is great, @pjpegion. It would probably have taken me a while to figure out where to find all of the necessary files to pull the appropriate data from!

I may want to generalize the interface for the script a bit, leaving the guts alone. For example, it would be nice to be able to give this script a lat/lon and for the script to find the appropriate FV3 grid point and grab that point's data. BTW, do you know of any good references for using the chgres script? I will likely need to know how to do this in the future to generate new orographic data for different resolutions.

Reply:
@grantfirl  The chgres script I'm familiar with only interpolates the spectral GFS initial conditions to the cubed-sphere grid.  I don't know which script creates the boundary condition files for a given resolution.  I recall Jeff Whitaker asking EMC to create these files when we wanted to run a different resolution than what was already available.